web_dynamic/static/scripts/3-hbnb.js

```diff
@@ -11,17 +11,17 @@ $(document).ready(function () {
     $('.amenities h4').text(nameAmenity.join(', '));
   });
 
-  $.get("http://localhost:5001/api/v1/status/", data => {
-    if (data.status == "OK") {
-      $('DIV#api_status').addClass("available");
+  $.get('http://0.0.0.0:5001/api/v1/status/', (res) => {
```

Comment:
you should leave localhost

Reply:
Request `http://0.0.0.0:5001/api/v1/status/`

Reply:
you change it in my code too

Reply:
I just make the scripts coherent since they are linked to each other

Reply:
they are going to ask for the reviewer to check if we are using the right URL or not